nemo_skills/mcp/servers/arxiv_tool.py

```
@@ -0,0 +1,756 @@
+    if cached := _cache_get(cache_key):
+        return cached
+
+    params: dict[str, Any] = {"mailto": POLITE_EMAIL}
+    if _openalex_api_key and not _key_exhausted:
+        params["api_key"] = _openalex_api_key
+    headers = {"User-Agent": USER_AGENT, "Accept": "application/json"}
+    url = f"{OPENALEX_BASE}/works/{norm}"
+    async with httpx.AsyncClient(headers=headers) as client:
+        try:
+            work = await _http_get_json(client, url, params=params)
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)